samples/snippets/src/main/java/com/example/datastore/AggregationQuerySample.java

```diff
@@ -0,0 +1,364 @@
+
+    // Save all the candidates
+    datastore.put(
+        Entity.newBuilder(candidate1Key).set("qualified", true).build(),
+        Entity.newBuilder(candidate2Key).set("qualified", false).build(),
+        Entity.newBuilder(candidate3Key).set("qualified", true).build()
+    );
+
+    EntityQuery selectAllCandidates = Query.newEntityQueryBuilder()
+        .setKind(kind)
+        .build();
+    // Creating an aggregation query to get the count of all candidates
+    AggregationQuery allCandidatesCountQuery = Query.newAggregationQueryBuilder()
+        .over(selectAllCandidates)
+        .addAggregation(Aggregation.count())
+        .addAggregation(Aggregation.count().as("total_count"))
+        .build();
+    // Executing aggregation query
+    AggregationResult allCandidatesCountQueryResult = Iterables.getOnlyElement(
+        datastore.runAggregation(allCandidatesCountQuery));
+
+    System.out.printf("Total candidates count is %d",
+        allCandidatesCountQueryResult.get("total_count")); // 3
+    System.out.printf("Total candidates (accessible from default alias) is %d",
+        allCandidatesCountQueryResult.get("property_1")); // 3
+
+    // [END datastore_count_aggregation_query_on_kind]
+  }
+
+  public void aggregationQueryAndCountAggregationWithLimit() {
+    // [START datastore_count_aggregation_query_with_limit]
+
+    // Instantiates a client
+    Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+
+    // The kind for the new entity
+    String kind = "Candidate";
+
+    Key candidate1Key = datastore.newKeyFactory().setKind(kind).newKey("candidate1");
+    Key candidate2Key = datastore.newKeyFactory().setKind(kind).newKey("candidate2");
+    Key candidate3Key = datastore.newKeyFactory().setKind(kind).newKey("candidate3");
+
+    // Save all the candidates
+    datastore.put(
+        Entity.newBuilder(candidate1Key).set("qualified", true).build(),
+        Entity.newBuilder(candidate2Key).set("qualified", false).build(),
+        Entity.newBuilder(candidate3Key).set("qualified", true).build()
+    );
+
+    EntityQuery selectAllCandidates = Query.newEntityQueryBuilder()
+        .setKind(kind)
+        .setLimit(2)
+        .build();
+    // Creating an aggregation query to get the count of all candidates
+    AggregationQuery allCandidatesCountQuery = Query.newAggregationQueryBuilder()
+        .over(selectAllCandidates)
+        .addAggregation(Aggregation.count().as("at_least"))
+        .build();
+    // Executing aggregation query
+    AggregationResult limitQueryResult = Iterables.getOnlyElement(
+        datastore.runAggregation(allCandidatesCountQuery));
+
+    System.out.printf("We have at least %d candidates", limitQueryResult.get("at_least")); // 2
+
+    // [END datastore_count_aggregation_query_with_limit]
+  }
+
+  public void aggregationQueryAndCountAggregationWithOrderBy() {
+    // [START datastore_count_aggregation_query_with_limit]
```

Comment:
This snippet tag is the same as the one for the previous snippet. Please update.

Reply:
Thanks @jlara310 for pointing this out, Fixed as part of https://github.com/googleapis/java-datastore/pull/871/commits/4a9c7baaace44aa617682c5f1e9a6fa0b50d47bf.